mmv1/templates/terraform/examples/privateca_certificate_generated_key.tf.erb

```diff
@@ -0,0 +1,91 @@
+# [START privateca_create_certificate_config]
```

Comment:
```suggestion
```

These are from a cloud.google.com integration that no longer exists, so we can safely remove them.